extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -84,12 +84,22 @@ export class ConsentStateManager {
    * @param {string=} consentStr
    */
   updateConsentInstanceState(instanceId, state, consentStr) {
+    console.log('update Consent Instance State');
```

Comment:
I think this is leftover debugging, should be removed? 😄

Reply:
oops